### PR TITLE
8278895: riscv: Rename Riscv64 to RISCV64

### PR DESCRIPTION
**Pull request by yhzhu20**
Rename Riscv64 to RISCV64 for consistence.

The related tests passed on QEMU:

- test/hotspot/jtreg/compiler/intrinsics/sha/cli/TestUseSHA1IntrinsicsOptionOnUnsupportedCPU.java
- test/hotspot/jtreg/compiler/intrinsics/sha/cli/TestUseSHA256IntrinsicsOptionOnUnsupportedCPU.java
- test/hotspot/jtreg/compiler/intrinsics/sha/cli/TestUseSHA512IntrinsicsOptionOnUnsupportedCPU.java
- test/hotspot/jtreg/compiler/intrinsics/sha/cli/TestUseSHAOptionOnUnsupportedCPU.java
- test/hotspot/jtreg/compiler/intrinsics/sha/cli/testcases/GenericTestCaseForOtherCPU.java
- test/hotspot/jtreg/compiler/intrinsics/sha/cli/testcases/GenericTestCaseForUnsupportedRISCV64CPU.java
- test/hotspot/jtreg/runtime/NMT/CheckForProperDetailStackTrace.java
- test/hotspot/jtreg/runtime/ReservedStack/ReservedStackTest.java
- test/jdk/jdk/jfr/event/os/TestCPUInformation.java
- test/lib-test/jdk/test/lib/TestMutuallyExclusivePlatformPredicates.java

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8278895](https://bugs.openjdk.java.net/browse/JDK-8278895): riscv: Rename Riscv64 to RISCV64

### Reviewers
 * @guotaiping1 (no known github.com user name / role)
 * [Fei Yang](https://openjdk.java.net/census#fyang) (@RealFYang - **Reviewer**)

### Contributors
 * Taiping Guo `<guotaiping1@huawei.com>`

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/riscv-port pull/33/head:pull/33` \
`$ git checkout pull/33`

Update a local copy of the PR: \
`$ git checkout pull/33` \
`$ git pull https://git.openjdk.java.net/riscv-port pull/33/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 33`

View PR using the GUI difftool: \
`$ git pr show -t 33`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/riscv-port/pull/33.diff">https://git.openjdk.java.net/riscv-port/pull/33.diff</a>

</details>

### CODE REVIEW

File: src/hotspot/cpu/riscv/vm_version_riscv.cpp [hunk 1]
```diff
@@ -179,6 +179,6 @@ void VM_Version::initialize_cpu_information(void) {
   _no_of_threads = _no_of_cores;
   _no_of_sockets = _no_of_cores;
   snprintf(_cpu_name, CPU_TYPE_DESC_BUF_SIZE - 1, "RISCV64");
-  snprintf(_cpu_desc, CPU_DETAILED_DESC_BUF_SIZE, "RV64%s", _features_string);
+  snprintf(_cpu_desc, CPU_DETAILED_DESC_BUF_SIZE, "RISCV64%s", _features_string);
```
Comment (yadongw) [inline src/hotspot/cpu/riscv/vm_version_riscv.cpp]: Keep it here. It's supposed to be RV64.

Comment (guotaiping1) [reply src/hotspot/cpu/riscv/vm_version_riscv.cpp]: 'RV64' will cause the test/jdk/jdk/jfr/event/os/TestCPUInformation.java case fail.

Comment (yhzhu20) [reply src/hotspot/cpu/riscv/vm_version_riscv.cpp]: Done.

**Review by guotaiping1 — APPROVED** [review]

**Review by RealFYang — APPROVED** [review]
Looks good. Thanks.

**Pull request closed without merging**